bucketVersionsStats.js

```diff
@@ -58,6 +62,21 @@ Optional environment variables:
 });
 const s3EndpointIsHttps = ENDPOINT.startsWith('https:');
 
+let _OLDER_THAN_TIMESTAMP;
+if (OLDER_THAN) {
```

Comment:
Instead of doing ad-hoc code, it could be more adequate to reuse the [parseOlderThan](https://github.com/scality/s3utils/blob/development/1.13/utils/parseOlderThan.js#L11) function, and extend it (if needed) to parse a number of seconds. This way, we both avoid surprises in users trying different tools with different expectations on OLDER_THAN as well as factorizing the code and making it more easily testable.

Personally I would prefer the spec to be an explicit `42 seconds` (with `42 second` accepted as well) instead of `42s`.

Reply:
If too complicated, we can just use the timestamp ISO8601

Reply:
ended up with ISO8601 + `<n> days` + `<n> seconds`